[8.12][ESS] Document Osquery Timeout setting (#4611)
elastic/security-docs: docs/osquery/invest-guide-run-osquery.asciidoc

+
TIP: Use <<osquery-placeholder-fields,placeholder fields>> to dynamically add existing alert data to your query. 

> .. Expand the **Advanced** section to set a timeout period for the query, and view or set {kibana-ref}/osquery.html#osquery-map-fields[mapped ECS fields] included in the results from the live query (optional).

Reviewer: ```suggestion
.. Expand the **Advanced** section to set a timeout period for the query, and view or set {kibana-ref}/osquery.html#osquery-map-fields[mapped ECS fields] to include in the results of the live query (optional).
```
If this makes sense to you

Author: Same comment as above

+

> NOTE: The default and minimum value for the **Timeout** field is 60 seconds (s). The maximum value is 9000 seconds.  

Author: ```suggestion
NOTE: The default and minimum value for the **Timeout** field is 60 seconds. The maximum value is 9000 seconds.  
```

. Go to the About section of the rule details page and click *Investigation guide*.
. Click the query. The Run Osquery pane displays with the *Query* field autofilled. Do the following:
.. Select one or more {agent}s or groups to query. Start typing in the search field to get suggestions for {agent}s by name, ID, platform, and policy.
.. Expand the **Advanced** section to set a timeout period for the query, and view or set {kibana-ref}/osquery.html#osquery-map-fields[mapped ECS fields] included in the results from the live query (optional).
+

> NOTE: The default and minimum value for the **Timeout** field is 60 seconds (s). The maximum value is 9000 seconds.  

Author: ```suggestion
NOTE: The default and minimum value for the **Timeout** field is 60 seconds. The maximum value is 9000 seconds.  
```
